.enb/make.js

```diff
@@ -5,6 +5,7 @@ var LIB_NAME = 'bem-components',
     path = require('path'),
     naming = require('bem-naming'),
     techs = require('./techs'),
+    page = require('./lib/page'),
```

Comment:
for me `pages` is more intuitive as this one node produces many pages (bundles) as its output